lib/_tls_wrap.js

```diff
@@ -281,7 +281,7 @@ TLSSocket.prototype._wrapHandle = function(handle) {
                 tls.createSecureContext();
   res = tls_wrap.wrap(handle, context.context, options.isServer);
   res._parent = handle;
-  res._reading = handle._reading;
+  res.reading = handle.reading;
```

Comment:
Is it going to matter that `handle.reading` is undefined for non-net.Socket handles?